exporter/opentelemetry-exporter-prometheus-remote-write/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+## Unreleased
```

Comment:
I think after [this](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/239) was merged, we are using a consolidated CHANGELOG file now. Please rebase.

Reply:
fixed!